### PR TITLE
Check qt version

### LINKED ISSUE #181: mix depends on Qt 5.5 (or latest) but does not report an error with lower qt versions
It appears that on windows webthree-helpers installs qt5.4.1 by default,
To my discovery Mix has issues with qt5,4,1 and does not work properly. (e.g unable to create project)

Mix should at least print notification message that it does not support specific qt version and could not work properly.

### PR DESCRIPTION
**Pull request by yann300**
fixes #181

### CODE REVIEW

File: qml/MainContent.qml [hunk 1]
```diff
@@ -55,6 +56,22 @@ Rectangle {
...
+		var qtversion = appService.qtVersion.split(".")
+		if (!(parseInt(qtversion[0]) >= 5 && parseInt(qtversion[1]) >= 3))
+			qtVersionDialog.open()
+
+	}
+
+	MessageDialog
+	{
+		id: qtVersionDialog
+		text: qsTr("The current installed Qt version is not compatible with Mix. Qt 5.3 minimun is needed. Usability might be degraded.")
```
Comment (bobsummerwill) [inline qml/MainContent.qml]: Little typo here - "minimun" should be "minimum".

**Comment by bobsummerwill** [comment]
What _is_ our minimum Qt version?
I know that OS X has 5.5.1 in Homebrew.
What kind of versions are in the various Ubuntu flavors and on Windows?

**Comment by chriseth** [comment]
As far as I know some html/js debugging feature requires Qt 5.5., but it might be that we only use that in alethzero.

**Comment by yann300** [comment]
yes we need 5.5 to build now because of AZ and the debugging feature.
But Mix should be runable with 5.3.

**Pull request merged**